packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -76,13 +53,32 @@ export function TransactionsTableRowAction({
     return isClaiming || isClaimingCctp || !isConfirmed
   }, [isClaiming, isClaimingCctp, isConfirmed])
 
-  const isRedeemButtonDisabled = useMemo(
-    () =>
-      typeof isConnectedToArbitrum !== 'undefined'
-        ? !isConnectedToArbitrum
-        : true,
-    [isConnectedToArbitrum]
-  )
+  const isConnectedToCorrectNetworkForRedeem = useMemo(() => {
```

Comment:
what's the difference bw `currentChainIsValid` and `isConnectedToCorrectNetworkForRedeem`? Both seem to return the same output for deposits. Maybe rename `currentChainIsValid` to `currentChainIsValidForAction` and make it generic, or have them both separate but with correct naming?

Reply:
`currentChainIsValid` is the old check that's for claiming only. the logic is a bit different because it also checks for whether it's deposit (for cctp).

I renamed it to `isConnectedToCorrectNetworkForClaim` for easier understanding